api/src/main/java/com/codeforcommunity/rest/subrouter/ProtectedReportRouter.java

```diff
@@ -19,87 +20,98 @@
 import static com.codeforcommunity.rest.ApiRouter.end;
 
 public class ProtectedReportRouter implements IRouter {
-    private static final String PREVIOUS_DAYS_QUERY_PARAM_NAME = "previousDays";
+  private static final String PREVIOUS_DAYS_QUERY_PARAM_NAME = "previousDays";
+  private final IProtectedReportProcessor processor;
 
-    private final IProtectedReportProcessor processor;
+  public ProtectedReportRouter(IProtectedReportProcessor processor) {
+    this.processor = processor;
+  }
 
-    public ProtectedReportRouter(IProtectedReportProcessor processor) {
-        this.processor = processor;
-    }
+  @Override
+  public Router initializeRouter(Vertx vertx) {
+    Router router = Router.router(vertx);
 
-    @Override
-    public Router initializeRouter(Vertx vertx) {
-        Router router = Router.router(vertx);
+    registerGetCommunityStats(router);
```

Comment:
🔧 Same as above here, we want a `ReportRouter` instead of a `ProtectedReportRouter`.